src/openms/source/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.cpp

```diff
@@ -127,6 +127,9 @@ namespace OpenMS
     {
       for (Size k = 0; k < picked_chroms[i].size(); ++k)
       {
+        // Skip peaks already "consumed" by previous iterations
+        if (picked_chroms[i][k].getIntensity() <= 1e-11) {continue; }
```

Comment:
Is it Zero? Note that floating point comparison against zero is fine if intensities are set to zero (and not a result of a calculation that involves rounding errors)

Reply:
Yes, when creating peak-group features (either by largest intensity or by widest peak), the assessed peaks are set to zero as a way to indicate the peak was already added/checked for a peak-group feature.